### DISCUSSION TITLE
What is your favorite Linux distribution?

### DISCUSSION THREAD

**Discussion by Hito50** [NAIST-SE/AttractingContributors#919] [General]
I often use Debian.
My repository is https://github.com/Hito50/Favorite-Linux-distribution-hobby-

**Comment by bconfessor** [comment]
I answered it (why don't you like Ubuntu??) !
I sent you a pull request! 
Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Comment by k-mizoguch1** [comment]
plz https://github.com/kzu000/Food-what-do-you-like
